contracts/contracts/eosio.system/src/eosio.system.cpp

```diff
@@ -478,13 +351,13 @@ EOSIO_DISPATCH( eosiosystem::system_contract,
      // native.hpp (newaccount definition is actually in eosio.system.cpp)
      (newaccount)(updateauth)(deleteauth)(linkauth)(unlinkauth)(canceldelay)(onerror)(setabi)
      // eosio.system.cpp
-     (init)(setram)(setramrate)(setparams)(setpriv)(setalimits)(setacctram)(setacctnet)(setacctcpu)(activate)
+     (init)(setram)(setramrate)(setparams)(setpriv)(setalimits)(activate)
```

Comment:
need to add (setminstake) here to make action visible:
(init)(setram)(setminstake)(setramrate)(setparams)(setpriv)(setalimits)(activate)